security/pkg/server/ca/server.go

```diff
@@ -101,6 +101,25 @@ func (s *Server) CreateCertificate(ctx context.Context, request *pb.IstioCertifi
 	return response, nil
 }
 
+// RootCertExpiration returns the remaining duration that root cert is still valid, if already expires, returns error.
+// ========================================
+// TODO: anyone who plan to support ROOT ROTATION FEATURE or MULTI ROOT FEATURE
+// ! PLEASE ENSURE THE ROOT CERT CAN BE ACCESSED SAFELY CONCURRENTLY !
+// ========================================
+func (s *Server) RootCertExpirationSeconds() float64 {
+	rb := s.ca.GetCAKeyCertBundle().GetRootCertPem()
+	cert, err := util.ParsePemEncodedCertificate(rb)
+	if err != nil {
+		fmt.Errorf("failed to parse the root cert: %v", err)
+		return -1
+	}
+	end := cert.NotAfter
+	if end.Before(time.Now()) {
+		log.Errorf("Expired Citadel Root found, x509.NotAfter %v, please checkout <TODO-LINK-TO-USER-GUIDE>", end)
```

Comment:
@wattli @wenchenglu are we gonna create a shorten link for this?

something like bit.ly/istio-rotate-cert?